test/app/dftb+/timedep/Furan-CIopt-LC/dftb_in.hsd

```diff
@@ -0,0 +1,50 @@
+Geometry = GenFormat {
+    <<< "geo.gen"
+}
+
+Driver = SteepestDescent{MaxSteps=0}
+Hamiltonian = DFTB {
+    SCC = Yes
+    SCCTolerance = 1e-8
+    MaxAngularMomentum = {
+        O = "p"
+        C = "p"
+        H = "s"
+    }
+    SlaterKosterFiles = Type2FileNames {
+        Prefix = "slakos/origin/ob2-1-1/shift/"
+        Separator = "-"
+        Suffix = ".skf"
+    }
+    RangeSeparated = LC {
+      Screening = Thresholded {
+        Threshold = 1.0e-16
+      }
+    } 
+}
+ExcitedState {
+    Casida {
+        NrOfExcitations = 10
+        StateOfInterest = 1
+        Symmetry = Singlet
+        Diagonaliser = Stratmann{}
+        StateCouplings = {1 2}
+	OptimiserCI = Bearpark{}
+    }
+}
+
+
+Options {
+    WriteAutotestTag = Yes
+}
+
+ParserOptions {
+    ParserVersion = 12
+}            
+
+Parallel {
+  # Allow OMP threads explicitely to test for hybrid parallelisation with
+  # MPI-binary. (Check the manual before using this in production runs!)
+  UseOmpThreads = Yes
```

Comment:
Redundant, as this is non-MPI at the moment.

Reply:
ok